### PR TITLE
adding trim to fields before sending them to backend

### PR DESCRIPTION
**Pull request by solidmartin**

### CODE REVIEW

File: src/connection/database/actions.js [hunk 1]
```diff
@@ -70,9 +70,11 @@ export function signUp(id) {
       additionalSignUpFields(m).forEach(x => {
         if (x.get('type') !== 'textinfo') {
           if (x.get('isExtra')) {
-            params.user_metadata.extraFields[x.get('name')] = c.getFieldValue(m, x.get('name'));
```
Comment (j-soliduslink) [inline src/connection/database/actions.js:73]: let's just put it into a single line
params.user_metadata.extraFields[x.get('name')] = c.getFieldValue(m, x.get('name')).trim() 
or
params.user_metadata.extraFields[x.get('name')] =
    c.getFieldValue(m, x.get('name')).trim()

File: src/connection/database/actions.js [hunk 2]
```diff
@@ -70,9 +70,11 @@ export function signUp(id) {
...
+            params.user_metadata.extraFields[x.get('name')] = c
```
Comment (j-soliduslink) [inline src/connection/database/actions.js:73]: is 'name' the combination of first name and last name? do we need to trim them separately?

**Review by j-soliduslink — APPROVED** [review]
LGTM

File: src/connection/database/actions.js (hunk 1 above)
Comment (solidmartin) [reply src/connection/database/actions.js:73]: I try, I think it was automatic

File: src/connection/database/actions.js (hunk 2 above)
Comment (solidmartin) [reply src/connection/database/actions.js:73]: Name is the name of the parameter, so they get trimmed separately. 

I think they use immutable.js in some places so I think x is an immutable object
 meaning x.get('name') is to be understood as x.name.

Comment (solidmartin) [reply src/connection/database/actions.js:73]:
```
additionalSignUpFields = Immutable.fromJS(additionalSignUpFields).map(x =>
      x.filter(y => y !== undefined)
    );
```
It's an immutable map

File: src/connection/database/actions.js (hunk 1 above)
Comment (solidmartin) [reply src/connection/database/actions.js:73]: Yes, it is formatted as part of the build step. Interesting configuration to come back to later...

**Comment by j-soliduslink** [comment]
btw, do you need to update the compiled version to our repo?

**Pull request merged**